src/content/self-hosted/administration/configuration.mdx

```diff
@@ -412,6 +413,7 @@ The daemonset automatically configures every node of your cluster to work better
 - `extraEnv`: Environment variables to be set on the daemonset containers.
 - `labels`: Labels to add to the daemonset pods.
 - `image`: Container image used by the daemonset pods.
+- `initContainers.privateCA.image`: Image of the init container used for the private CA. Defaults to `busybox`
```

Comment:
@pchico83 @jpf-okteto something more specific to comment about this?

Reply:
Nope, that's fine